src/future_jiffies_patch.c

```diff
@@ -14,6 +14,12 @@
 #include <linux/timer.h>
 
 #define BUFFER_SIZE 11
+/*
+ * why hard-coded?
+ * why 11?
+ * please, double check ULONG_MAX in /usr/include/limits.h
```

Comment:
Data types limits is the most popular topic to make mistakes, it should be checked very carefully.
Especially if we write modules which should work on different arch-es (e.g. on x86_32/x86_64/aarch64/etc)